### ISSUE TITLE
Wrong location attributed when manually add location for a photo without GPS EXIF data

### ISSUE THREAD

**Issue by eugensim** [mapillary/mapillary_issues#2091]
Just added a photo manually on the map, but after the upload the location is wrong (300-400 meters south-west), not the one that I've located on the map. 
Link: https://www.mapillary.com/app/user/streetview_romania?lat=44.38334305555556&lng=26.116670555555555&z=17&pKey=DecFGfP6ax84hgrzWME5dQ&focus=photo but correct location is here 
![image](https://cloud.githubusercontent.com/assets/8439643/17102147/e8bdf116-5280-11e6-8c80-97cc56cf1471.png)

/Webupload
/Chrome last version

**Comment by ToeBee** [comment]
Pretty sure I just ran into the same bug. 

I just uploaded several sequences from a bicycle tour I did in Seattle after SOTM-US. These were GoPro images that I geotagged externally. My Garmin was in my pocket and drifted more than usual (probably also due to the urban canyon effect) so I manually adjusted the position of a lot of the pictures in the new "Review" screen on the manual upload page.

It looks like all the pictures that I manually changed the location of in a given sequence ended up piled on top of each other in one location over a mile away from where the sequence actually is. See this screenshot of the sequence as it ended up on the map:

![seattle_sequence](https://cloud.githubusercontent.com/assets/348948/17274531/5823ad2e-56ab-11e6-86dd-b061e584b531.png)

All the sequences I've uploaded today are affected. Is there any way to recover the locations or do these sequences just need to be deleted so I can re-upload them?

Link to the first picture in the sequence: https://www.mapillary.com/app/?pKey=AVVlisJtoDmmeTKfOgxTSA

**Comment by gyllen** [comment]
This is now fixed. Thanks for reporting. If you want us to remove sequences so you can upload please paste them here, otherwise the solution would be to use edit to correct the positions. Once agaion extremly sorry for this bug we were not rounding coordinates correctly in uploader :/

**Comment by ToeBee** [comment]
I just tried editing the sequences but there are so many pictures on top of each other that it seems to get the editor all confused after I move 10-15 of them and it just stops working. Plus I can't really tell which picture goes in which order. So yeah... I'm going to need these sequences to be hidden/deleted:

`UcaW05Qkspa90tc1rwFjng`
`ByFb_IfwiXrs2NGmC8-g4w`
`hOIY5b8Ijr9n7r7CB9siOw`
`smMmQZqRnO_gfb22JgYCew`
`TYxlNtxeOOLmMXj2FCWBRg`

Also, did this affect the sequence editor as well as the uploader? I submitted two edits (I think it was two) during SOTM-US. They haven't been approved yet as far as I can tell but I'm wondering if they also moved images to a single location. They are sequences `fkygKo6qzJwdwclVDLCP_g` and `cvJiL4CcF9onk-X751lE-g`
